server.js

```diff
@@ -27,9 +41,47 @@ const app = express()
 app.use(cors())
 app.use(bodyParser.json())
 
+
 // Start defining your routes here
 app.get('/', (req, res) => {
-  res.send('Hello world')
+  res.send('Hello welcome to my golden globe api!')
+})
+
+app.get('/nominees', async (req, res) => {
+  
+  const getNominations = await Nomination.find(req.query)
+
+  if (getNominations) {
+    res.json(getNominations)
+  } else {
+    res.status(404).json({ error: 'There is no nomination like this'})
+  }
+})
+
+app.get('/nominees/:year/:category', async (req, res) => {
+  const { year, category } = req.params
+  let filteredNominees = await Nominee.find(
+    { 
+      year_award: year, 
+      category: category
+    })
+  res.json(filteredNominees)
+})
+
+//ONE ENDPOINT 
+app.get('/nominee/:year/:category/win=true', async (req, res) => {
+  const { year, category } = req.params
+  let filteredNominees = await Nominee.find(
+    {
+      year_award: year,
+      category: category,
+      win: true
+    });
+  if (filteredNominees) {
+    res.json(filteredNominees)
+  } else {
+    res.status(404).json({ error: 'Not found' })
+  }
 })
```

Comment:
The logic seems fine, but you should not hard code a query param into the path like that. Did you get this endpoint to work? You could add the win as a query param under the previous endpoint (since this is a duplicate besides the win=true).